data/food/export.py

```diff
@@ -176,23 +200,30 @@ def find_id(dbname, activity):
             if attribute in process:
                 del process[attribute]
 
-    print("Computing corrected impacts (etf-c, htc-c, htn-c)...")
-    with open(IMPACTS, "r") as f:
-        processes = with_corrected_impacts(json.load(f), processes)
 
-    # export ingredients
-    with open(INGREDIENTS, "w") as outfile:
-        json.dump(ingredients, outfile, indent=2, ensure_ascii=False)
-        # Add a newline at the end of the file, to avoid creating a diff with editors adding a newline
-        outfile.write("\n")
-    print(f"\nExported {len(ingredients)} ingredients to {INGREDIENTS}")
+if __name__ == "__main__":
+    setup_environment()
 
-    # display impacts that have changed
-    display_changes("id", oldprocesses, processes)
+    # keep the previous processes with old impacts
+    oldprocesses = load_json(CONFIG["PROCESSES_FILE"])
+    activities = load_json(CONFIG["ACTIVITIES_FILE"])
 
-    # export processes
-    with open(PROCESSES, "w") as outfile:
-        json.dump(list(processes.values()), outfile, indent=2, ensure_ascii=False)
-        # Add a newline at the end of the file, to avoid creating a diff with editors adding a newline
-        outfile.write("\n")
-    print(f"Exported {len(processes)} processes to {PROCESSES}")
+    ingredients = create_ingredient_list(activities)
+    compute_land_occupation(activities)
```

Comment:
`<purist talking>`I'm never comfortable with non-pure functions that modify data in place that lead to difficult bugs, I prefer returning new frozen data to be assigned to a different variable. That's a deformation from functional programming :-/  I should have used a frozendict from the beginning.`</purist talking>`